src/Lean/Data/JsonRpc.lean

```diff
@@ -38,6 +38,8 @@ inductive ErrorCode where
   | methodNotFound
   | invalidParams
   | internalError
+  | workerCrashed
```

Comment:
Let's put these error codes at the end, below "Lean-specific codes below.", and rename `workerCrashed` -> `workerExited`, `stackOverflow` -> `workerCrashed`.

Reply:
Ok :)